README.md

```diff
@@ -18,6 +18,24 @@ Data can be examined by species, contributing laboratory, methodology, brain reg
 
 It is to be noted that many other scientific fields (Astronomy, Agriculture, Bioinformatics, Pharmaceutical industry, ...) are in need of such a technology. Consequently, Blue Brain Nexus core technology is being developed to be **agnostic of the domain** it might be applied to.
 
+# Getting started
+Running locally
+The easiest way to run BlueBrain Nexus locally is to use docker-compose.
+
+*Prerequisites*
```

Comment:
Same here. I would put `##Prerequisites ` or `###Prerequisites ` instead of `*Prerequisites*`